tf-modules/data-persistence/README.md

```diff
@@ -0,0 +1,45 @@
+# Data Persistence
+
+This module deploys data persistence resources, including DynamoDB tables and an Elasticsearch instance (which is optional).
+
+## Deployment
+
+1. Copy the .tfvars sample file: `cp terraform.tfvars.sample terraform.tfvars`
+2. Populate the sample file with values that apply to your AWS environment (see configuration variables section below).
+3. Deploy this module: `terraform apply`
+
+NOTE: Terraform will ignore the `aws_profile` config variable if you have static credentials or environment variables set, see the [AWS Provider page](https://www.terraform.io/docs/providers/aws/index.html#authentication).
+
+## Included resources
+
+- DynamoDB tables:
+  - `AccessTokensTable`
+  - `AsyncOperationsTable`
+  - `CollectionsTable`
+  - `ExecutionsTable`
+  - `FilesTable`
+  - `GranulesTable`
+  - `PdrsTable`
+  - `ProvidersTable`
+  - `RulesTable`
+  - `SemaphoresTable`
+  - `UsersTable`
+- Elasticsearch domain (with optional VPC configuration)
+- Cloudwatch alarm for Elasticsearch node count
+
+**Please note**: All created resource names will be prefixed with the value of your `prefix` variable plus a hyphen (e.g. `prefix-AccessTokensTable`).
+
+## Configuration
+
+Configuration variables are shown in [`terraform.tfvars.sample`](./terraform.tfvars.sample), and are explained below.
+
+name|required|default|description
```

Comment:
I opted for a table to document the options. I liked that it didn't repeat `terraform.tfvars.sample`, structured the output a bit better, and seemed more natural for providing better explanations of each config value than comments in a code block. Thoughts?

Reply:
I don't feel strongly one way or another. I don't really care for presenting the same information a bunch of different ways, but that's probably my own preference as some people might prefer a table because they think differently. I'll grant that adding notes/context is easier in a table.